monero-outreach-docs/translations/es/storyboard/wallet_quickstart_es.md

```diff
@@ -0,0 +1,56 @@
+
+Al escoger entre estos, ten en cuenta que la mayoría tiene una opción extra en la cual — siendo la excepción The Edge Wallet — pueden ser configurados para conectarse ya sea a un servidor remoto de terceros o a un servidor que ejecutes con un nodo. Esto se ilustra en la tabla.
+
+## Monederos Oficiales
+
++ [getmonero.org/es/downloads](https://www.getmonero.org/es/downloads)  
+Si deseas ejecutar tu propio nodo completo, una buena elección es el monedero Monero GUI o CLI oficial. El primero provee una interfaz gráfica de usuario, y el último una interfaz textual.
+
+## Monederos Móviles
+
++ [cakewallet.io](https://cakewallet.io/)  
+Cake Wallet, un monedero ligero de protección de claves-de-visualización, está disponible para dispositivos iOS.
+
++ [monerujo.io](https://www.monerujo.io/)  
+Monerujo, un monedero ligero de protección de claves-de-visualización, está disponible para dispositivos Android.
+
++ [edge.app](https://edge.app/)  
+Edge Wallet está disponible para ambos Android y iOS. Es un monedero ligero de compartir claves de visualización.
+
+## Monederos Web *(Optimizados para Conveniencia)*
+
++ [mymonero.com](https://mymonero.com/)  
+MyMonero es un monedero ligero, basado en navegador. También está disponible como una app de escritorio y app de iOS.
+
+## Monederos Hardware *(Optimizados para Seguridad)*
+
++ [ledger.com](https://shop.ledger.com/?r=92d74dc2847a)  
+Ledger es un dispositivo altamente seguro que guarda tu semilla mnemónica. No aplica.
+
++ [trezor.io](https://trezor.io/)  
+Trezor es una alternativo de dispositivo altamente seguro que guarda tu semilla mnemónica.
+
+Para un monedero hardware que provea seguridad extra en almacenar y transmitir Monero incluso si el dispositivo primario se ve comprometido, ambos Ledger, [ledger.com](https://shop.ledger.com/?r=92d74dc2847a), y Trezor, [trezor.io](https://trezor.io/), son buena consideración. (Nótese que Ledger trabaja ahora con monederos, mientras que el soporte para monedero en Trezor está en progreso).
+
+Dejar que tus preferencias de recursos y privacidad conduzcan en la elección de tu monedero maximizará tu experiencia en Monero. La riqueza de Monero en opciones de monedero — desde una rápida app MyMonero hasta un monedero autoritario de nodo completo Monero CLI combinado con una Ledger o Trezor — permite a Monero satisfacer las diversas necesidades de sus usuarios.
```

Comment:
I would change this a bit, not sure what you think:

"_**La riqueza/variedad en las opciones de monederos en Monero**_ - desde una rápida app MyMonero hasta un monedero _**autorizado**_...".

Reply:
_variedad_ sounds better.

About _autoritario_,  took me a while to get to that sentence while reading the original phrase. Used because it sounds more like a adjective using "wallet" as a subject, but what do you think? _Autoritario_ is less used, of course.

Reply:
The word "autoritario" in our social context, obviously this differs in regions, means and it clearly defines "authority" for something but by someone and not a thing, e.g. "él es autoritario".

Reply:
Well, yes, and now that I see one of the translations as an adjective is, indeed, _autorizado_. But since there's another world for that adjective (authorized) I used that one.

Reply:
But I do get your point. The sentence is a little confusing when reading it several times. I will go with your preference.